dias/analyzers/source_spectra_analyzer.py

```diff
@@ -213,9 +232,8 @@ def setup(self):
         self.data_index.commit()
 
 	# initalise run metric
-        #self.run_metric.labels(source=src).set(0)
-
-        #self.logger.info("Sources: {}".format(self.source_transits))
+        self.flux_metric = self.add_data_metric(
+            "flux", labelnames=['frequency', 'source', 'pol'], unit='Jy')
```

Comment:
Nice! I think the unit shouldn't be abbreviated, though: https://prometheus.io/docs/practices/naming/